File: 37/Squash.py
from Campi import Campi

class Squash(Campi):
    def __init__(self, codice, tipo, nomeCampo, larghezza, lunghezza, costoOrario,altezzaInMetri,piano) -> None:
        super().__init__(codice, tipo, nomeCampo, larghezza, lunghezza, costoOrario)
        self.altezzaInMetri = altezzaInMetri
        self.piano = piano
 
    def __str__(self) -> str:
        return self.tipo+"\t" + self.nomeCampo+"\t" + str(self.codice)+"\t" + str(self.larghezza)+"\t" + str(self.lunghezza) + "\t-\t" + "-" + str(self.altezzaInMetri)+"\t" + str(self.piano) +"\t"+ str(self.costoOrario)
    

File: Sim02 -Campi Con Split/src/Squash.java
/*
 * @ Perrotta Marco
 * 
 * Struttura figlia di Campo 
 * 
 */

public class Squash extends Campo {
    float altezza;
    int piano;

    public Squash(int codice, String sport, String nomeCampo, float larghezza, float lunghezza, float costo_orario,
            float altezza, int piano) {
        super(codice, sport, nomeCampo, larghezza, lunghezza, costo_orario);
        this.altezza = altezza;
        this.piano = piano;
    }

    public String toString() {
        return sport + " "+ nomeCampo + " "+codice +" "+ larghezza +" "+ lunghezza + "- " + "- " + altezza+" " + piano+" " + costo_orario;
    }

}
